django/core/serializers/json.py

```diff
@@ -108,6 +109,8 @@ def default(self, o):
             return r
         elif isinstance(o, decimal.Decimal):
             return str(o)
+        elif isinstance(o, uuid.UUID):
+            return str(o)
```

Comment:
I am not aware of any Django coding style guideline for this case, but I tend to use the isinstance(o, ( ... )) style only for classes that are closely related (like tuple and list) and be explicit if that's not the case to increase readability (all different cases are one below the other) 
However, I'm happy to change it if you think that would be more proper.